lib/blueberry_rails/app_builder.rb

```diff
@@ -54,9 +55,7 @@ def create_shared_flashes
 
     def create_application_layout
       remove_file 'app/views/layouts/application.html.erb'
-      template 'layout.html.slim.erb',
-        'app/views/layouts/application.html.slim',
-        force: true
+      copy_file 'layout.html.slim', 'app/views/layouts/application.html.slim'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.